### ISSUE TITLE
Incorrect UI in QS media player

### ISSUE THREAD

**Issue by madenerrgy** [OxygenCobalt/Auxio#490]
### Describe the Bug/Crash

![Screenshot_20230625-144509_Auxio](https://github.com/OxygenCobalt/Auxio/assets/77206030/27dd650e-afe1-4283-b7ae-1eb9a2461866)
This is what the shuffle button looks like.

![Screenshot_20230625-144511_Auxio](https://github.com/OxygenCobalt/Auxio/assets/77206030/4a9cc0b9-41b3-41b5-97af-c24e9b239dda)

This is how the disabled "shuffle" button looks like.
We can conclude that in fact nothing is visible, and you are constantly confused whether this button is enabled or not.
My friend also confirms that he constantly faces this unpleasant moment.

### Describe the intended behavior

It seems to me that this is best implemented by Google itself:

![Screenshot_20230625-144520_Pixel_Launcher](https://github.com/OxygenCobalt/Auxio/assets/77206030/441608a6-8a84-45a4-9fd3-ab570b9f2912)
Just a dot that says shuffle is on.
![Screenshot_20230625-144521_Pixel_Launcher](https://github.com/OxygenCobalt/Auxio/assets/77206030/fede6a99-ff47-4419-a77e-2851daf77eb7)
And accordingly when it is turned off.

### What android version do you use?

Android 13

### What device model do you use?

Pixel 7a

### Bug report

No need, visual bug.

### Duplicates

- [X] I have checked the [Troubleshooting](https://github.com/OxygenCobalt/Auxio/wiki/Troubleshooting) page.
- [X] I have checked this issue for duplicates.
- [X] I have checked that this issue occurs on the [lastest version](https://github.com/OxygenCobalt/Auxio/releases).
- [X] I agree to the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by illdeletethis** [comment]
not a fan of the dot
would prefer 2 straight horizontal arrows for regular, crossed arrows for shuffle

**Comment by OxygenCobalt** [comment]
The problem is that I am unable to use a clear indicator style that is actually consistent between components. 

- Tacking a dot onto the vector icon scales in a way that makes it so small as to be useless. I am then forced to write my own view code that is impossible to port to the widgets.
- Tidal works around this by using custom icons which are also just slightly translated upwards, which I find infuriatingly inconsistent.
- Using a faded state indicates that the button is disabled, not off.
- Using different arrow directions for each state is unexpected visual communication for most users.

So, I just communicate the state by changing thickness. I did notice though that the shuffle icon wasn't as thick as it could be. I've tried to increase it as much as I could and ended up with this:
![Screenshot_2023-06-25_16-43-02](https://github.com/OxygenCobalt/Auxio/assets/65370175/cd8ddfec-2d54-4a5a-8e27-c79d5227654f)

This is as far as I will go. I've gone through this whole goose chase before and settled on this, and it's extremely unlikely I will change it.
